src/sgame/sg_bot_ai.cpp

```diff
@@ -871,7 +871,7 @@ AINodeStatus_t BotActionFight( gentity_t *self, AIGenericNode_t *node )
 	BotAimAtEnemy( self );
 	BotMoveInDir( self, MOVE_FORWARD );
 
-	if ( inAttackRange || self->client->ps.weapon == WP_PAIN_SAW )
+	if ( inAttackRange )
```

Comment:
Dunno. Bots already have aim speed dependent on their skill. But I agree they could use a random modifier to decide wether to shoot or not. This could apply to *any* weapon though. This change notably does not impacts their efficiency against enemy, but it makes it less frustrating for allies as no more stupid death because cut by "friend".
To have watched this patch in action, it honestly is not that aimbotty, as they tend to keep blaster for too long (since they shoot with blaster, they can't swap weapon before cooldown time). It might even make this better.
Need to think more on that topic.

Reply:
I honestly think this arbitrary `2 * PAINSAW_RANGE` idea would not bring much.

Reply:
Guess you're right on that. I'll see to change that.